src/radchallenge.c

```diff
@@ -0,0 +1,210 @@
+/*
+ * radchallenge.c - a sample c program showing how to embed the configuration of a radius and to challenge
+ * client, using the FreeRADIUS Client Library without an external configuration and dictionary file .
+ */
+
+#include <stdlib.h>
+#include <sys/types.h>
+#include <syslog.h>
+#include <radcli/radcli.h>
+#include <string.h>
+
+#define AUTH_PORT_PASSWORD ":1812:testing123"
+#define ACCT_PORT_PASSWORD ":1813:testing123"
+#define SERVER_ADDR "localhost"
+
+int
+main (int argc, char **argv)
+{
```

Comment:
I think it makes sense to include it as a test in tests/ so that this functionality is tested.

Reply:
Is it possible to run such test?
If yes, how can I do it?

I have no idea how tests should look like. I will try to add one with rc_dict_addattr

Reply:
Yes these tests run when you commit in github via travis. You can see their run:
https://travis-ci.org/github/radcli/radcli/builds/719301652